packages/injectable/ogre-tools-injectable.d.ts

```diff
@@ -145,7 +145,12 @@ declare module '@ogre-tools/injectable' {
     transient: ILifecycle;
   };
 
-  export function createContainer(...getRequireContexts: any[]): DiContainer;
+  export interface RequireContext {
```

Comment:
I wonder if there already exists a type for this.

Reply:
Maybe but I didn't want to add `webpack-dev` as a dependency.